armory/docker/management.py

```diff
@@ -96,7 +96,9 @@ def exec_cmd(self, cmd: str, user=""):
                 if inner_output == scenarios.END_SENTINEL:
                     sentinel_found = True
 
-        if sentinel_found:
+        # if we're not running a config (eg armory exec or launch)
+        #  we don't expect the sentinel to be printed, so its absence is not an error
+        if sentinel_found or not expect_sentinel:
             logger.info("Command exited cleanly")
```

Comment:
Let's only log this when sentinel is found. If we're not expecting a sentinel, we have no way of knowing if the command was successful. Currently, this happens:
```
armory exec pytorch -- ls1
2020-10-21 17:07:16 mac0831.local armory.docker.management[48795] INFO ARMORY Instance ab26a8c893 created.
2020-10-21 17:07:16 mac0831.local armory.eval.evaluator[48795] INFO Running bash command: ls1
OCI runtime exec failed: exec failed: container_linux.go:349: starting container process caused "exec: \"ls1\": executable file not found in $PATH": unknown
2020-10-21 17:07:17 mac0831.local armory.docker.management[48795] INFO Command exited cleanly
2020-10-21 17:07:17 mac0831.local armory.eval.evaluator[48795] WARNING Shutting down container
```
Perhaps this should go into a three way branch?
not expected --> return 0, no logging
sentinel --> return 0, log
no sentinel --> return 1, log